pyccel/codegen/printing/ccode.py

```diff
@@ -1350,6 +1356,9 @@ def get_c_type(self, dtype):
             val_type = self.get_c_type(dtype.value_type).replace(' ', '_')
             i_type = f'{container_type}_{key_type}_{val_type}'
             self.add_import(Import(f'stc/{container_type}', AsName(VariableTypeAnnotation(dtype), i_type)))
+            self.add_import(Import(f'{stc_extension_mapping["stc/" + container_type]}',
+                                   AsName(VariableTypeAnnotation(dtype), i_type),
+                                   ignore_at_print=True))
```

Comment:
@jalalium didn't you add something so that both of these includes are imported automatically?
I don't like the idea that we have to do this manually. It seems like a good way to regularly forget the extensions import (as seems to have happened here)

Reply:
@EmilyBourne Indeed, In #2086, `_print_header` now automatically adds extension headers along with the stc headers. However, I should also have removed the old calls to both imports; I guess this is the cause of this confusion. I need to open a PR for this.

Reply:
What do you want to do for this PR then? Merge as is? Remove this import?

Reply:
Ok. I have created #2156 to track this. I am going to go ahead and merge this PR.